graphstorm-processing/graphstorm_processing/config/config_conversion/gconstruct_converter.py

```diff
@@ -97,7 +97,21 @@ def _convert_feature(feats: list[dict]) -> list[dict]:
 
                 if gconstruct_transform_dict["name"] == "max_min_norm":
                     gsp_transformation_dict["name"] = "numerical"
-                    gsp_transformation_dict["kwargs"] = {"normalizer": "min-max", "imputer": "mean"}
+                    gsp_transformation_dict["kwargs"] = {"normalizer": "none", "imputer": "none"}
```

Comment:
This is the conversion of the min-max transformation

```suggestion
                    gsp_transformation_dict["kwargs"] = {"normalizer": "min-max", "imputer": "none"}
```